README.md

```diff
@@ -1,26 +1,30 @@
 # Django PostgreSQL Extensions
-
 This package tries to expose functionality from PostgreSQL to django
 applications.
 
-
-Python 2.6 or greater is required.
-
+Python 2.7 is required.
 
 ## Features
-
-- Support for the full text search engine of PostgreSQL.
 - Support for the COPY command.
 - Support for ARRAYs. Out of the box support for INT and TEXT arrays. The
   operators @> (contains), <@ (is contained by) and && (overlaps with) are
   supported.
 - Case-insensitive variants of `CharField` and `SlugField`.
 
-
-## Running the tests
-
-Just run
-
-    django-admin.py test --settings=djangopg.test_settings
-
-from the root directory.
+### Building Docker container
+1. In order to run tests (using Docker containers) you need download the following Python packages:
+```
+pip download --no-cache-dir -r requirements/tests.txt --dest pip-cache
+```
+Note: `coverage`, `filelock`, `psycopg2`, `scandir` and `tox` you need `tar.gz` instead of `whl` version.
+
+2. Build container:
+```
+docker-compose build
+```
+
+### Tests
+Run test against real PostgreSQL instance, using Docker:
+```
+docker-compose run --rm tox
+```
```

Comment:
I got rid of the known issue after writing a test, I suspect that the lookups thingy we removed was causing the issue.